config/locales/de.yml

```diff
@@ -920,7 +920,7 @@ de:
     order_state:
       address: Adresse
       awaiting_return: erwartet Erstattung
-      canceled: Abgebrochen
+      canceled: Storniert
```

Comment:
@JDutil don't ask Google, ask your German friends. ;)

It's totally correct in this context and can be safely merged. Thanks for asking.